### PR TITLE
Sorted extensions to make diff of generated code easier to read.

### PR DESCRIPTION
**Pull request by rajder**
The purpose is to minimize the length of the diff rows and the number of diff rows in the generated code.

### CODE REVIEW

**Comment by Dav1dde** [comment]
Thanks for the pull, I took the diff and slightly changed it (newline in commandline doesn't work for copy pasting), I also found a bug where extensions where listed more than once.

**Comment by rajder** [comment]
Yeah that's why I added the backslash for line continuation in bash. But on windows that needs to be ^ so not perfect :)

File: glad/lang/common/generator.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def commandline(self):
         generator = '--generator="{}"'.format(self.NAME)
         specification = '--spec="{}"'.format(self.spec.NAME)
         loader = '' if self.has_loader else '--no-loader'
-        extensions = '--extensions="{}"'.format(','.join(self.extension_names))
+        extensions = '--extensions="\\\n            {}"'.format(',\\\n            '.join(self.extension_names))
```
Comment (rajder) [inline glad/lang/common/generator.py]: backslash for line continuation in bash.

**Comment by Dav1dde** [comment]
Yes, but as you stated doesn't work on Windows and in my opinion there is no additional value in formatting it, since there is already the formatted (and copy pasteable) list above.

**Comment by rajder** [comment]
True, the command line blob can be ignored when viewing the diff.

**Pull request closed without merging**